CMake/resolve_dependency_modules/arrow/CMakeLists.txt

```diff
@@ -38,7 +38,8 @@ if(VELOX_ENABLE_ARROW)
       -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
       -DARROW_BUILD_STATIC=ON
       -DThrift_SOURCE=${THRIFT_SOURCE}
-      -DCMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH})
+      -DCMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}
+          -DCMAKE_POLICY_VERSION_MINIMUM=3.5 )
```

Comment:
Sadly this won't work due to arrow using external_project for most of the dependencies.